project2/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=ZCOOL+QingKe+HuangYou|Khand" rel="stylesheet">
+</head>
+<title>Card Specimen of Spider-Man</title>
+
+<body>
+    <div class="rapper">
+        <figure>
+            <img src="images/Spiderman.jpg" alt="spider-man with black background" width="200px" height="200px">
+        </figure>
+    </div>
+    <div class="context">
+        <h1 class="title" style="width: 360px;">Spider-Man: Into the Spider Verse</h1>
+        <div class="rate">
+            <span>8.6/10 </span>
+            <span class="length">
+                1h 56m</span>
+            <span class="playbutton">
```

Comment:
Since this is a button and should perform an action for the user (for example, link to the movie), this should be an `a` element, not a `span`. Note that when you don't have an actual location to link to, you can replace the `href` value with the pound sign/hashtag (`#`) and then the browser will still treat this link a link:
```suggestion
            <a class="playbutton" href="#">
```